examples/acoustic_example.py

```diff
@@ -3,15 +3,16 @@
 from Acoustic_codegen import Acoustic_cg
 from containers import IGrid, IShot
 
+
 dimensions = (50, 50, 50)
 model = IGrid()
 model0 = IGrid()
 model1 = IGrid()
 model.shape = dimensions
 model0.shape = dimensions
 model1.shape = dimensions
-origin = (0., 0.)
-spacing = (20., 20.)
+origin = (0., 0., 0.)
+spacing = (20., 20., 20.)
```

Comment:
This is fixed by #115 so we should first merge that, and then rebase this on top of it